pkg/controller/networkpolicy/store/group.go

```diff
@@ -121,6 +125,11 @@ func GroupKeyFunc(obj interface{}) (string, error) {
 	return k8s.NamespacedName("", group.Name), nil
 }
 
+// ObjectRefKeyFunc knows how to get the indexer key of a object reference in a Group.
+func ObjectRefKeyFunc(name, namespace string) string {
```

Comment:
can we reuse k8s.NamespacedName func?

Reply:
Unfortunately we can't. The podReference or eeReference is simply a struct comprised on namespace and name, and does not embed ObjectMeta.

Reply:
actually k8s.NamespacedName also uses string and does not rely on ObjectMeta. Maybe I should be clear. The func im asking to use is part of antrea `pkg.k8s` and not really the upstream func, unless i  missed something

```
// NamespacedName generates the conventional K8s resource name,
// which connects namespace and name with "/".
func NamespacedName(namespace, name string) string {
	if namespace == "" {
		// Cluster scoped resources will contain empty namespace.
		return name
	}
	return namespace + "/" + name
}
```

Reply:
Sorry I thought you meant metaNamespaceKeyFunc. Yes we should reuse k8s.NamespacedName. Thanks for pointing it out. Updated.